guides/common/modules/proc_performing-a-full-backup.adoc

```diff
@@ -43,6 +43,13 @@ You can use the `{foreman-maintain} backup` command to back up remote databases.
 
 You can use both online and offline methods to back up remote databases, but if you use offline method, the `{foreman-maintain} backup` command performs a database dump.
 
+.Backing up to a remote NFS share
+
+To enable {Project} to save the backup to the NFS share, make sure that the `root` user of your {ProjectServer} or {SmartProxyServer} can write to the NFS share.
```

Comment:
```suggestion
To enable {Project} to save the backup to an NFS share, make sure that the `root` user of your {ProjectServer} or {SmartProxyServer} can write to the NFS share.
```

Not sure about `a`/`an`, but there is no specific NFS share at that point, so `the` seems wrong :)